cmd/github-labels/README.md

```diff
@@ -0,0 +1,75 @@
+* [Overview](#overview)
+* [Generating the combined labels database](#generating-the-combined-labels-database)
+* [Checking and summarising the labels database](#checking-and-summarising-the-labels-database)
+    * [Show labels](#show-labels)
+    * [Show categories](#show-categories)
+    * [Check only](#check-only)
+    * [Full details](#full-details)
+
+# Overview
+
+The Kata Project uses a number of GitHub repositories. To allow issues and PRs
+to be handled consistently between repositories a standard set of issue labels
+are used. These labels are stored in YAML format in the master [labels
+database template](labels.yaml.in). This file is human-readable and
+machine-readable and self-describing (take a look at the file for the
+introductory description).
+
+Additionally, each repository can contain a set of additional
+(repository-specific) labels which are stored in a top-level YAML template
+file called `labels.yaml.in`.
+
+Expanding the templates and merging the two databases describes the full set
+of labels a repository uses.
+
+# Generating the combined labels database
+
+The combined labels database is generated by running the
```

Comment:
Lines 27-30 suggested rewrite (unless I changed the meaning): 
You can run the `github_labels.sh` script with the `generate` parameter to generate the combined labels database. This passes the repository in order to generate the combined labels database and the name of a file to write the combined database:

Reply:
I've changed your wording slightly to:

> You can run the `github_labels.sh` script with the `generate` argument to
create the combined labels database. The additional arguments specify the
repository (in order to generate the combined labels database) and the name of
a file to write the combined database: